docs/cloud/provision/README.md

```diff
@@ -567,3 +575,9 @@ Due to the burstable CPU class, CPU shares are limited, this results in the foll
 | **M32**  | Production                     | 12M                   | 1 TB            | 250                      |
 | **M64**  | Production                     | 30M                   | 2 TB            | 500                      |
 | **M128** | Production                     | 62M                   | 4 TB            | 500                      |
+
+### Disk Types
```

Comment:
If this section exists it should probably go through the motions of explaining the disk types for each provider, even though there's not really any choice on which to use outside of AWS. 

Another idea: this whole `Disk Types` section could be removed from this PR for now and a subsequent one could add it and explain all the types (including the deprecated GP2).